app/views/prison/visits/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :header do %>
+  <%= t('.title') %>
+<% end %>
+
+<%= render(partial: 'people_box') %>
+
+<% if @booking_response.unavailable? %>
+  <h2>
+    <%= t('.unavailable', status: @booking_response.processing_state) %>
```

Comment:
This should be localised - we can't rely on `processing_state` being English (or even human-friendly). I think the right thing here is to have each condition as a separate translation:

```yaml
withdrawn: "The visitor has withdrawn this request"
rejected: "This visit request has been rejected"
```

etc.